pkg/minikube/logs/logs.go

```diff
@@ -186,5 +190,15 @@ func logCommands(r cruntime.Manager, bs bootstrapper.Bootstrapper, length int, f
 	}
 	cmds[r.Name()] = r.SystemLogCmd(length)
 	cmds["container status"] = cruntime.ContainerStatusCommand()
+
+	cfg, err := config.Load(viper.GetString(config.ProfileName))
+	if err != nil && !config.IsNotExist(err) {
```

Comment:
You can just say 
```
if err != nil
```
here

Reply:
okay, thanks for the review! Are the failing checks unrelated to my changes? I couldn't really tell.